src/java.xml/share/classes/javax/xml/transform/OutputKeys.java

```diff
@@ -31,6 +31,7 @@
  * output properties from a Transformer or Templates object.
  * <p>All the fields in this class are read-only.</p>
  *
+ * @spec https://www.w3.org/TR/xslt xslt cover page - W3C
  * @see <a href="http://www.w3.org/TR/xslt#output">
```

Comment:
The pages for XSLT and XPath at W3C are organized differently from the days when this javadoc was created. The "latest version" now points to the "cover page". Could you change the spec to the following? 
      https://www.w3.org/TR/1999/REC-xslt-19991116 XSL Transformations (XSLT) Version 1.0
      
The @spec points to the general spec while @see also a specific section (similar situation as other classes in the package), if we want to keep @see ref here, it would be: 
    https://www.w3.org/TR/1999/REC-xslt-19991116#output